tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -260,29 +265,51 @@ def _setup_node(self) -> None:
         self._log.info("Resource Group: %s", self.context.vm.resource_group)
         self._log.info("")
 
+        self._install_tools_on_node()
+
         if self.context.is_vhd:
             self._log.info("Using a VHD; will not install the test Agent.")
         else:
             self._install_agent_on_node()
 
+    def _install_tools_on_node(self) -> None:
+        """
+        Installs the test tools on the test node
+        """
+        self.context.ssh_client.run_command("mkdir -p ~/bin")
+
+        tools_path = self.context.test_source_directory/"orchestrator"/"scripts"
```

Comment:
now we copy orchestrator/scripts to the VMs in order to execute those scripts over SSH

nag, if you need scripts for your tests, probably we should create a similar subdirectory under the tests directory and copy it here

Reply:
copy to where?` ~bin` path

Reply:
Yes, copy to ~/bin.

 '~' means $HOME for the current user (waagent on the automation pipeline, or whatever user is passed as parameter to the runbook for dev runs).